test/test_spectral_ops.py

```diff
@@ -157,20 +173,39 @@ def test_reference_1d(self, device, dtype, op):
 
     @skipCPUIfNoFFT
     @onlyNativeDeviceTypes
-    @dtypes(torch.float, torch.double, torch.complex64, torch.complex128)
+    @toleranceOverride({
+        torch.half : tol(1e-2, 1e-2),
+        torch.chalf : tol(1e-2, 1e-2),
```

Comment:
is this standard for the rest of our tests for `half` and `chalf`?

Reply:
These for consistent for fft. For other test like `test_complex_half_reference_testing` we use default value for `self.assertEqual` which defaults to `atol=1e-3, rtol=1e-5`